modules/etcd-certificates.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * authentication/understanding-certificates.adoc
+
+[id="etcd-certificates_{context}"]
+= etcd certificates
+
+
+TBD:
```

Comment:
@hexfusion Can you please help me build out this section? The customer is asking for the information outlined here.

Reply:
@hexfusion Can you please provide some information on etcd certificates by end of week? We are needing the following:

* The purpose
* File path
* Default expiration term
* How to set custom expiration term
* How to specify the expiration date of all certificates used in OpenShift when installing OpenShift
* Which services use it
* How to update/extend it
* Will certificates that are about to expire be automatically renewed by the operator?

Reply:
@retroflexer Can you help provide some of this information for @ahardin-rh?

Reply:
**Note**: the information here between <= 4.2/4.3 and >=4.3/4.4 will differ due to the etcd-operator (whenever it's introduced).  So keep that in mind when cherry-picking the content.

Reply:
@retroflexer Can you please provide the information?

Reply:
Sorry, missed this one earlier. I will try to respond by EOB. Thanks.

Reply:
Apologies for a late response. Here are some of the answers. Let me know if you have any questions:
https://docs.google.com/document/d/1uY0eNPaQDwp9_7MIW4PBT9Nnx7dg5wDpSVXtlFJUdX8/edit?usp=sharing